.moai/specs/SPEC-V3R3-BRAIN-001/acceptance.md

```
@@ -0,0 +1,338 @@
+```
+
+---
+
+## Scenario 4: SPEC Decomposition List Parseable by `/moai plan --from-brain`
+
+**Maps to**: REQ-BRAIN-004, REQ-BRAIN-007 (downstream integration)
+
+**Given**:
+- A completed brain workflow has produced `.moai/brain/IDEA-001/proposal.md`
+- The proposal.md contains a `### SPEC Decomposition Candidates` section with the following entries:
+  ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)